docs/docs/homepage/features.tsx

```diff
@@ -0,0 +1,107 @@
+import { translate } from "@docusaurus/Translate";
+import BasedOnThoroughResearch from "../../src/components/icons/BasedOnThoroughResearch";
+import CensorshipResistance from "../../src/components/icons/CensorshipResistance";
+import HighThroughput from "../../src/components/icons/HighThroughput";
+import IsomorphicStateChannels from "../../src/components/icons/IsomorphicStateChannels";
+import LowFees from "../../src/components/icons/LowFees";
+import React from "react";
+import LowLatency from "../../src/components/icons/LowLatency";
+
+export const FeatureList = [
+  {
+    title: translate({
+      id: "homepage.featureList.lowLatency.title",
+      message: "Low Latency",
+    }),
+    icon: <LowLatency />,
+    description: translate({
+      id: "homepage.featureList.lowLatency.description",
+      message:
+        "Transaction finality is only bounded by Head network latency, resulting in near-instant settlement",
+    }),
+    tagLine: translate({
+      id: "homepage.featureList.lowLatency.perk",
+      message: "Fast and cheap transactions",
```

Comment:
Can we just keep these "taglines" empty? They are _currently redundant_ and although they look good, we likely don't need a tagline on such a small description above:

![image](https://github.com/user-attachments/assets/433b7abb-294b-4bb3-a487-56f3b2255295)